evalml/pipelines/components/estimators/classifiers/lightgbm_classifier.py

```diff
@@ -22,17 +22,15 @@ class LightGBMClassifier(Estimator):
         "n_estimators": Integer(10, 100),
         "max_depth": Integer(0, 10),
         "num_leaves": Integer(2, 100),
-        "min_child_samples": Integer(1, 100),
-        "bagging_fraction": Real(0.000001, 1),
-        "bagging_freq": Integer(0, 1)
+        "min_child_samples": Integer(1, 100)
     }
     model_family = ModelFamily.LIGHTGBM
     supported_problem_types = [ProblemTypes.BINARY, ProblemTypes.MULTICLASS]
 
     SEED_MIN = 0
     SEED_MAX = SEED_BOUNDS.max_bound
 
-    def __init__(self, boosting_type="gbdt", learning_rate=0.1, n_estimators=100, max_depth=0, num_leaves=31, min_child_samples=20, n_jobs=-1, random_state=0, bagging_fraction=0.9, bagging_freq=0, **kwargs):
+    def __init__(self, boosting_type="gbdt", learning_rate=0.1, n_estimators=100, max_depth=0, num_leaves=31, min_child_samples=20, n_jobs=-1, random_state=0, **kwargs):
```

Comment:
We remove both bagging args from the base `__init__` parameters because bagging isn't accepted for `boosting_type=goss` and would result in a fatal error.